### ISSUE TITLE
Ui-tooltip destroy in Ui-Fab on mobile devices

### ISSUE THREAD

**Issue by marksuurland** [JosephusPaye/Keen-UI#394]
When the ui-fab component is used with an additional tooltip (ui-tooltip) for example:

```
    <ui-fab
      v-if="showFab"
      class="list__button"
      icon="add"
      color="accent"
      @click="createNewContact"
      tooltip="Add new contact"
      tooltipPosition="left middle"
      openTooltipOn="hover"
    ></ui-fab>
```
This causes the following error, when used on mobile devices:

```
Uncaught TypeError: Cannot read property 'parentNode' of null
    at DropInstance.open (keen-ui.js?bc59:2242)
    at eval (keen-ui.js?bc59:2185)
```
After some brief research, the problem seems to be on the .destroy() in tether-tooltip, changing it into remove() does solve the error however then the tooltip is still being shown.

**Comment by JosephusPaye** [comment]
Thanks for reporting, I'll look into this.

I have been working on switching from Tether to [Popper.js](popper.js.org). That'll probably fix this, and other Tether-related problems.

**Comment by EmilMoe** [comment]
I have been using v-tippy with great success, @JosephusPaye

**Comment by JosephusPaye** [comment]
Fix released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
